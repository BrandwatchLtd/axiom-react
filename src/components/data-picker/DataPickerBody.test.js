import React from 'react';
import renderer from 'react-test-renderer';
import { DataPickerBody } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <DataPickerBody { ...props }>
      Lorem Ipsum
    </DataPickerBody>
  );
}

describe('DataPickerBody', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});


