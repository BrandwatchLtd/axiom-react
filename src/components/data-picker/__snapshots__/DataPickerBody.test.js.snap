// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataPickerBody renders with defaultProps 1`] = `
<div
  className="ax-card__content ax-card__content--separator-solid ax-card__content--size-medium"
>
  Lorem Ipsum
</div>
`;
